### DISCUSSION TITLE
How to use Attend and Excite with DreamBooth+LoRA / Custom Diffusion?

### DISCUSSION THREAD

**Discussion by daeunni** [huggingface/diffusers#7285] [Q&A] [answered]
Hi, I am wondering how to combine `Attend and Excite` pipeline with DreamBooth+LoRA / Custom Diffusion setting. 
I figured out StableDiffusionAttendAndExcitePipeline doesn't have `load_lora_weights` function. 

Can anyone help me?

**Comment by tolgacangoz** [comment, marked as answer]
Hi @daeunni!
Try to inherit from `LoraLoaderMixin` to be able to use `load_lora_weights()`. Add `LoraLoaderMixin` [here](https://github.com/huggingface/diffusers/blob/25caf24ef90fc44074f4fd3712f6ed5a1db4a5c3/src/diffusers/pipelines/stable_diffusion_attend_and_excite/pipeline_stable_diffusion_attend_and_excite.py#L173).
```diff
class StableDiffusionAttendAndExcitePipeline(DiffusionPipeline, StableDiffusionMixin, TextualInversionLoaderMixin,
+                                            LoraLoaderMixin):
```
